eng/Versions.external.props

```diff
@@ -10,13 +10,17 @@
 
   <!-- Automatically include these assemblies in all test projects -->
   <ItemGroup Condition="$(MSBuildProjectName.EndsWith('Tests'))">
-     <PackageReference Include="Moq" Version="4.16.1" />
-     <PackageReference Include="xunit" Version="2.6.3" />
-     <PackageReference Include="xunit.runner.visualstudio" Version="2.5.5">
+     <PackageReference Include="Moq" Version="4.20.70" />
```

Comment:
Why are we updating these dependencies? We should be particularly careful with this one.

Reply:
These are development dependencies only, so I see no reason why we shouldn't keep up to date. The issue with Moq has been resolved.